Frameworks/UIKit/XamlControls.mm

```diff
@@ -27,7 +27,9 @@
 // Button
 ////////////////////////////////////////////////////////////////////////////////////
 WXCButton* CreateButton() {
-    ComPtr<IInspectable> inspectable(XamlCreateButton());
+    ComPtr<IInspectable> inspectable;
+    // Use Attach for transfer-ownership semantics, else we +1 and leak
+    inspectable.Attach(XamlCreateButton());
```

Comment:
Your real problem is the XamlCreateButton api design.  If it returned HRESULT and an out pointer, you have to try hard to leak it.  With the current model, it is easy to mess it up.